<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Sharing Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="pages">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>660</width>
      <height>600</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="mainPage">
     <widget class="QPushButton" name="btnBack">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>40</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>‚ùÆ</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbPath">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>511</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: green;</string>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>49</y>
        <width>640</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSignOut">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>70</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign Out</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRefresh">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>560</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCreateFolder">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>560</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Folder</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUpload">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>560</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Upload</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDownload">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>560</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDelete">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>560</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCreateGroup">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>560</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Group</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnJoinGroup">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>560</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Join Group</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="loginPage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>70</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>220</y>
        <width>70</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtUsername">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>170</y>
        <width>260</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtPassword">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>220</y>
        <width>260</width>
        <height>30</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSignIn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>290</y>
        <width>240</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign In</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSignUp">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>340</y>
        <width>240</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign Up</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
